Assets/LeapMotion/Modules/InteractionEngine/Examples/Common Example Assets/Scripts/IgnoreCollisionsInChildren.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Leap.Unity.Examples {
+
+  /// <summary>
+  /// This utility script scans through its Transform's children and tells PhysX to
+  /// ignore collisions between all pairs of Colliders it finds. This is particlarly
+  /// useful, for example, for interfaces, where buttons shouldn't collide with other
+  /// buttons.
+  /// 
+  /// This is not the recommended strategy in general: It is much more optimal for your
+  /// application to put interface objects on a layer and disable self-collision for that
+  /// layer in your Physics settings (Edit / Project Settings / Physics).
+  /// </summary>
+  [AddComponentMenu("")]
```

Comment:
Hmm I'm not sure we should go so far as preventing people from adding this script to their gameObjects.  I think it would be very useful to have something like this for prototyping and development before one goes all the way to create multiple layers.